### PR TITLE
Zynq support

### PR DESCRIPTION
**Pull request by Himanshu-xilinx**
* initial Zynq support
* it does not support interrupts

### CODE REVIEW

**Comment by chvamshi-xilinx** [comment]
ok to test

File: src/runtime_src/tools/scripts/petalinux.sh [hunk 1]
```diff
@@ -59,9 +59,17 @@ echo " * Setup PetaLinux: $PETALINUX_LOCATION"
 
 # dsa_build.sh will create ${PLATFORM_NAME}/ directory for petalinux project
 if [ -z $BSP ]; then
-	${XRT_REPO_DIR}/src/runtime_src/tools/scripts/peta_build.sh ${ORIGINAL_DIR}/dsa_build/${PLATFORM_NAME}.dsa
+	if [ -d "${XRT_REPO_DIR}/src/platform/${PLATFORM_NAME}/config.sh" ]; then
```
Comment (mamin506) [inline src/runtime_src/tools/scripts/petalinux.sh]: This is not necessary, if the config.sh exist in the .dsa directory, the peta_build.sh would source config.sh.

**Comment by maxzhen** [comment]
ok to test

File: src/runtime_src/tools/scripts/petalinux.sh (hunk 1 above)
Comment (Himanshu-xilinx) [reply src/runtime_src/tools/scripts/petalinux.sh]: code updated

**Comment by NaAl-Xlnx** [comment]
retest this please

**Comment by NaAl-Xlnx** [comment]
retest this please

**Comment by maxzhen** [comment]
retest this please

**Pull request merged**